src/NHibernate/Linq/GroupJoin/GroupJoinAggregateDetectionVisitor.cs

```diff
@@ -34,6 +36,7 @@ public static IsAggregatingResults Visit(IEnumerable<GroupJoinClause> groupJoinC
 
 		protected override Expression VisitSubQuery(SubQueryExpression expression)
 		{
+			//Visit the entire query model?
```

Comment:
I don't see the need to visit the entire query model as aggregated expressions should always be located in the selector.

Reply:
So shouldn't this comment be removed?

Reply:
I removed it as I cannot think of any scenario where the whole query model would need to be visited.